spinta/backends/nobackend/commands/changes.py

```diff
@@ -0,0 +1,18 @@
+from typing import AsyncIterator
+
+from spinta import commands
+from spinta.components import Context, Property, Model, DataItem
+from spinta.backends.nobackend.components import NoBackend
+
+
+@commands.create_changelog_entry.register(Context, (Model, Property), NoBackend)
+def create_changelog_entry(
+    context: Context,
+    model: (Model, Property),
+    backend: NoBackend,
+    *,
+    dstream: AsyncIterator[DataItem],
+) -> AsyncIterator[DataItem]:
+    # FileSystem properties don't have change log, change log entry will be
+    # created on property model backend.
+    return dstream
```

Comment:
This command is called from write operations, we don't need to define it from `NoBackend`, because we can't do any write operations if the first place.